compaction/compaction_manager.cc

```diff
@@ -1201,13 +1215,14 @@ class regular_compaction_task_executor : public compaction_task_executor, public
 }
 
 void compaction_manager::submit(table_state& t) {
-    if (_state != state::enabled || t.is_auto_compaction_disabled_by_user()) {
```

Comment:
How about `t.is_auto_compaction_disabled_by_user()`?

Reply:
fixed in v2 ([30ad31e](https://github.com/scylladb/scylladb/commit/30ad31ef01855f55ac5308829d49d5ea0dc560c7))